builtInFunctions/esdtDataStorage.go

```diff
@@ -131,6 +131,9 @@ func (e *esdtDataStorage) getESDTNFTTokenOnDestinationWithAccountsAdapterOptions
 		Type:  uint32(core.Fungible),
 	}
 	marshaledData, _, err := accnt.AccountDataHandler().RetrieveValue(esdtNFTTokenKey)
+	if core.IsGetNodeFromDBError(err) {
```

Comment:
let's add unit tests for all the new if branches

Reply:
Added.